bag_unpacker/src/main/scala/uk/ac/wellcome/platform/archive/bagunpacker/services/Unpacker.scala

```diff
@@ -62,77 +66,64 @@ case class Unpacker(s3Uploader: S3Uploader)(implicit s3Client: AmazonS3) {
 
     s"$archiveLocation could not be downloaded"
 
-//    cause.getStatusCode match {
-//      case 403 => s"access to $archiveLocation is denied"
-//      case 404 => s"$archiveLocation does not exist"
-//      case _   => s"$archiveLocation could not be downloaded"
-//    }
+    //    cause.getStatusCode match {
```

Comment:
delete?

Reply:
Oh yeah, we commented this out as a reminder to go back and do it properly!

Done.